descwl_shear_sims/wcs/sewcs.py

```diff
@@ -41,12 +44,19 @@ def make_se_wcs(
     Galsim WCS of the single exposure
     """
 
+    if dither_size is not None and not dither:
+        raise ValueError("dither_size is set but dither is False")
+
     if dither:
         # do a small offset of the origin
         assert rng is not None
-        dither_range = 0.5
-        off = rng.uniform(low=-dither_range, high=dither_range, size=2)
+
+        if dither_size is None:
+            dither_size = 0.5
```

Comment:
I thjink this default should be visible in the `make_sim` declaration

Reply:
Fixed!